scss/util/_color.scss

```diff
@@ -92,35 +107,39 @@
   @return scale-color($color, $lightness: $scale);
 }
 
-/// Get color from foundation-palette
+/// Get color from foundation-colors
 ///
-/// @param {key} color key from foundation-palette
+/// @param {key} color key from foundation-colors
 ///
-/// @returns {Color} color from foundation-palette
+/// @returns {Color} color from foundation-colors
 @function get-color($key) {
```

Comment:
What also needs to be considered after looking on my own code, that this is a breaking change! So there needs to be a migration warning in Release Notes for 6.4
Or maybe we should make this backward compatible by doing
```SCSS
@if map-has-key($foundation-colors, $key) {
  @return map-get($foundation-colors, $key);
} @else if map-has-key($foundation-palette, $key) {
  @warn 'returning color from $foundation-palette which is deprecated behavior and will be remove in future versions';
  @return map-get($foundation-palette, $key);
} @else {
  @error '...'
}